pkgs/applications/version-management/transcrypt/default.nix

```diff
@@ -32,6 +30,16 @@ stdenv.mkDerivation rec {
     chmod +x $out/bin/transcrypt-depspathprefix
   '';
 
+  doInstallCheck = true;
+  installCheckPhase = ''
+    if [[ "$($out/bin/transcrypt --version)" == "${pname} ${version}" ]] ; then
+      echo "${pname} passed install check"
+    else
+      echo "${pname} failed install check"
+      return 1
+    fi
+  '';
```

Comment:
May use `testers.testVersion`

Reply:
moved to passthru version testing in https://github.com/NixOS/nixpkgs/pull/247309/commits/3b4cceb8783108734460bb899f8d774d10bb729b

Thanks for the heads up